apps/builder/app/builder/features/sidebar-left/panels/pages/page-utils.test.ts

```diff
@@ -364,6 +365,87 @@ describe("isSlugAvailable", () => {
   });
 });
 
+describe("isPathAvailable", () => {
+  const pages = createDefaultPages({
```

Comment:
please write visual structure in comment like
```
// root
//   folder
//     page
```

Reply:
I think we need a function that takes a nested description that is easier to read than the current one, these comments get outdated quickly